isovar/variant_sequence_in_reading_frame.py

```diff
@@ -208,6 +224,30 @@ def count_mismatches(reference_prefix, cdna_prefix):
                 reference_prefix, cdna_prefix))
     return sum(xi != yi for (xi, yi) in zip(reference_prefix, cdna_prefix))
 
+def count_mismatches_after_variant(reference_suffix, cdna_suffix):
+    """
+    Computes the number of mismatching nucleotides between two cDNA sequences after a variant locus.
+
+    Parameters
+    ----------
+    reference_suffix : str
+        cDNA sequence of a reference transcript after a variant locus
+
+    cdna_suffix : str
+        cDNA sequence detected from RNAseq after a variant locus
+    """
+    
+    len_diff = len(cdna_suffix) - len(reference_suffix)
+    if len_diff > 0:
+        # if the reference is shorter than the read, the read runs into the intron - these count as
+        # mismatches. pad the reference suffix if necessary with underscores
+        reference_suffix += '_' * len_diff
```

Comment:
Not a fan of modifying the cDNA sequence with extra characters. Why not just add `max(0, len_diff)` to the sum below?